src/middleware.js

```diff
@@ -24,16 +24,20 @@ export default store => {
 
         rpc
           .call(action.name, args)
-          .then(result =>
-            store.dispatch(
-              actions.fetchSuccess({
-                name: action.name,
-                args,
-                id,
-                result,
-              }),
-            ),
-          )
+          .then(result => {
+            console.log(result)
```

Comment:
Should we remove this debugging line? If you find this logging useful for you, maybe add an option to the middleware like `{ debug: true }`?